Sources/RichTextKit/Coordinator/RichTextCoordinator+Subscriptions.swift

```diff
@@ -6,38 +6,58 @@
 //  Copyright © 2022-2023 Daniel Saidi. All rights reserved.
 //
 
-#if iOS || macOS || os(tvOS)
+#if iOS || macOS || tvOS
 import SwiftUI
 
 extension RichTextCoordinator {
 
-    /// Make the coordinator subscribe to context changes.
-    func subscribeToContextChanges() {
+    func subscribeToUserActions() {
+        richTextContext.userInitiatedActionPublisher.sink { [weak self] action in
+            switch action {
+            case .changeStyle(let style, let newValue):
+                self?.setStyle(style, to: newValue)
+            case .triggerAction(let action):
+                self?.handle(action)
+            case .shouldPasteImage(let insertion):
+                // TODO: Only one paste action and switch in other function?
+                self?.pasteImage(insertion)
+            case .shouldPasteImages(let images):
+                self?.pasteImages(images)
+            case .shouldPasteText(let text):
+                // TODO: Is this even used?
+                self?.pasteText(text)
+            case .shouldSelectRange(let range):
+                // TODO: Is this even used?
+                self?.setSelectedRange(to: range)
+            case .shouldSetAttributedString(let attributedString):
+                // TODO: Is this even used?
+                self?.setAttributedString(to: attributedString)
```

Comment:
can any of the TODOs be resolved?

Reply:
As pointed out by @danielsaidi those are public API, which is not directly used in project but can be used whenever someone else is using this library. Deleted those TODOs.